### DISCUSSION TITLE
Solana

### DISCUSSION THREAD

**Discussion by GoldTaro** [Huniko-Team/community#190] [Q&A] [answered]
What is Solana and how does it differ from other blockchains like Ethereum?

**Comment by Tvenus** [comment, marked as answer]
Solana is a high-performance blockchain supporting smart contracts and decentralized applications. 
It uses a unique consensus mechanism called Proof of History (PoH) combined with Proof of Stake (PoS) to achieve high throughput and low latency.
 Unlike Ethereum, which faces scalability issues and high gas fees, Solana can handle thousands of transactions per second at a fraction of the cost.
